ex6/ex6.c

```diff
@@ -22,6 +22,23 @@ and `clock_gettime()` should work just fine.
 int main()
 {
     // Your code here
+    uint64_t diff;
```

Comment:
Does this code compile for any of you? In my case, it states that `uint64_t` is an unknown type name. Keep in mind that many C types are unavailable without including specific header files...